### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by rodrigondec** [dcrbrdev/JackBot#6]
Create the official README and instructions for our project.

We must describe:
- What and why was this project created (motivation)
  - Link to [Ticket Splitting Buyer documentation](https://github.com/matheusd/dcr-split-ticket-matcher)
- Development setup (with possible ways of running the projec: python and docker)
  - how to setup python virtual env
  - how to setup docker + docker-compose
  - how to create your bot on telegram with BotFather
  - how to run the project
- Specify this project as Open Source and ask for help maintaining/developing 
  - Contribute guide (how to contribute)

### References
- https://github.com/jerimumhs/JerimumBot
- https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github
- https://github.com/daal-imd-dev/integraTI-api/blob/master/CONTRIBUTING.md
- https://github.com/thoughtbot/factory_bot_rails/blob/master/CONTRIBUTING.md
- https://github.com/atom/atom
- https://github.com/rails/rails
- https://github.com/matheusd/dcr-split-ticket-matcher

**Comment by som23ya** [comment]
Is this issue open? Can i work on it?

**Comment by rodrigondec** [comment]
Yes. It's open and your help is welcomed 😁

I'll describe exactly what would be god to be done

**Comment by som23ya** [comment]
Hey, I am really new to open source and contributions, so i would be
eagerly waiting for guidance to the project. I have intermediate python
knowledge though :D

Somya Upadhyay

**Comment by rodrigondec** [comment]
@som23ya 
Ok, I created another issue for documentation #9. Feel free if you prefer to do the other issue.

Are you familiar with the basic of open source contributing? (forks, pull requests, reviews)

**Comment by som23ya** [comment]
Yes i do have basic knowledge of github and commands. I will look up the
other issue too.

Somya Upadhyay

**Comment by rodrigondec** [comment]
feel free to do just some part of the issue too. I believe the most important now is do `Development setup`
